Stratis.Bitcoin/Features/Consensus/PosConsensusValidator.cs

```diff
@@ -14,17 +14,30 @@ namespace Stratis.Bitcoin.Features.Consensus
     /// TODO
     /// </summary>
     /// <remarks>
-    /// These are the criteria for a new block to be accepted as a valid POS block:
+    /// These are the criteria for a new block to be accepted as a valid POS block at version 3 of the protocol, 
+    /// which has been active since 6 August 2016 07:03:21 (Unix epoch time > 1470467000). All timestamps 
+    /// are Unix epoch timestamps with seconds precision.
     /// <list type="bullet">
-    /// <item>New block's timestamp has to be strictly greater than previous block's timestamp.</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
-    /// <item>TODO</item>
+    /// <item>New block's timestamp ('BlockTime') MUST be strictly greater than previous block's timestamp.</item>
+    /// <item>Coinbase transaction's (first transaction in the block with no inputs) timestamp MUST be inside interval ['BlockTime' - 15; 'BlockTime'].</item>
```

Comment:
I would avoid putting hard limits, those can be configured in the protocol options, you can specify that as an example (i.e in stratis block time mast be ['BlockTime' - 15; 'BlockTime'])